client/query/oracle.go

```diff
@@ -0,0 +1,56 @@
+package query
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	oracletypes "github.com/ojo-network/ojo/x/oracle/types"
+)
+
+func (c *Client) OracleQueryClient() oracletypes.QueryClient {
```

Comment:
Godocs for these functions please
Also if it's only used within the Client object:
```suggestion
func (c *Client) oracleQueryClient() oracletypes.QueryClient {
```

Reply:
https://github.com/ojo-network/ojo/pull/103/commits/848898abe4ab5984b1783cbc326e772492a5d55c